mobile/src/pages/ImageCreate.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import { Text, Pressable, View, ScrollView, SafeAreaView } from "react-native";
+import Button from "../components/Button";
+import Card from "../components/Card";
+import Icon from "react-native-vector-icons/Ionicons";
+import BottomNavBar from "../components/BottomNavBar";
+import TopNavBar from "../components/TopNavBar";
+import SelectableChip from "../components/SelectableChip";
+
+export const imageStyles = [
+    "Muppet",
+    "Realistic",
+    "Cartoon",
+    "Pencil Sketch",
+    "Line Drawing",
+    "Oil Painting",
+    "Futuristic",
+    "Noire",
+    "Watercolor",
+    "Pixelated",
+    "Mosaic",
+    "Retro",
+    "Abstract",
+    "Minimalist",
+    "Vintage",
+    "Gothic",
+    "Graffiti",
+    "Vaporwave",
+    "Anime",
+    "Manga",
+    "4K",
+    "3D",
+    "HDR",
+    "Detailed",
+];
+
+export default function ImageCreate({ navigation }) {
+    const [search, setSearch] = useState("");
+
+    return (
+        <View className="flex-1">
+        <SafeAreaView className="flex-1">
+            <View className="android:mt-16 mb-4 ml-2 flex flex-row items-center">
+                <Pressable>
+                    <Icon
+                        size={30}
+                        name="arrow-back-outline"
+                        onPress={() => navigation.navigate("FeedPage")}
+                    />
+                </Pressable>
+                <View className="grow">
+                    <TopNavBar icon="color-wand-outline" className="m-2" value={search} onChangeText={setSearch} />
+                </View>
+            </View>
+            <ScrollView>
+                <View className="flex flex-row flex-wrap justify-evenly">
+                    {imageStyles.map(style =>
+                        <SelectableChip
+                            key={style}
+                            className="m-2 max-w-fit"
+                            label={style}
+                            selected={false}
+                            onSelect={() => { }}
+                        />
+                    )}
+                </View>
+                <View className="flex mt-4 items-center">
+                    <Button name="Create" />
+                </View>
+                <View className="flex-row items-center">
+                    <Text className="m-3 mr-2 text-2xl">Recently created</Text>
+                    <Icon size={25} name="timer-outline"/>
+                </View>
+                <View className="">
+                    <Card></Card>
+                </View>
+            </ScrollView>
+        </SafeAreaView>
+        <BottomNavBar />
```

Comment:
ok this is kinda mb for this one, I made added props to BottomNavBar without adding appropriate docs for it,
BottonNavBar must pass in the `navigation` object from ImageCreate's prop. So that line will just look like:
`<BottomNavBar navigation={navigation}/>`
similar to how its done in feedPage.

Reply:
This is not your fault; it's the natural consequence of this merge.